android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -486,53 +441,55 @@ class StripeSdkModule(private val reactContext: ReactApplicationContext) : React
           promise.resolve(res)
         }
 
-        override fun onError(error: Exception) {
-          promise.resolve(createError("Failed", error))
+        override fun onError(e: Exception) {
+          promise.resolve(createError("Failed", e))
         }
       }
     )
   }
 
   @ReactMethod
-  fun handleCardAction(paymentIntentClientSecret: String, promise: Promise) {
-    val activity = currentActivity as ComponentActivity
-    if (activity != null) {
-      handleCardActionPromise = promise
-      stripe.handleNextActionForPayment(activity, paymentIntentClientSecret)
-    }
+  @SuppressWarnings("unused")
+  fun handleNextAction(paymentIntentClientSecret: String, promise: Promise) {
+    paymentLauncherFragment.clientSecret = paymentIntentClientSecret
+    paymentLauncherFragment.promise = promise
+    paymentLauncherFragment.paymentLauncher.handleNextActionForPaymentIntent(paymentIntentClientSecret)
   }
 
-  private fun payWithWeChatPay(paymentIntentClientSecret: String, appId: String) {
-    val activity = currentActivity as ComponentActivity
-
-    activity.lifecycleScope.launch {
-      stripe.createPaymentMethod(PaymentMethodCreateParams.createWeChatPay()).id?.let { paymentMethodId ->
-        val confirmPaymentIntentParams =
-          ConfirmPaymentIntentParams.createWithPaymentMethodId(
-            paymentMethodId = paymentMethodId,
-            clientSecret = paymentIntentClientSecret,
-            paymentMethodOptions = PaymentMethodOptionsParams.WeChatPay(
-              appId
-            )
-          )
-        stripe.confirmPayment(activity, confirmPaymentIntentParams)
-      }
-    }
-  }
+// TODO: Uncomment when WeChat is re-enabled in stripe-ios
```

Comment:
nope, we're okay here since the only reference to this method is commented out- https://github.com/stripe/stripe-react-native/blob/master/android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt#L545

Reply:
cool, thanks